examples/counter-metrics/counter_metrics.bal

```diff
@@ -0,0 +1,56 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/io;
+import ballerina/observe;
+
+@http:ServiceConfig { basePath: "/online-store-service" }
+service<http:Service> OnlineStoreService bind { port: 9090 } {
+
+    //Create a counter as a global varaible in the service with optional field description.
+    observe:Counter globalCounter = new("total_orders", desc = "Total quantity required");
+
+    @http:ResourceConfig {
+        path: "/make-order"
+    }
+    makeOrder(endpoint caller, http:Request req) {
+        //Incrementing the global counter defined with the default value 1.
+        globalCounter.increment();
+
+        //Create a counter with simply a name.
+        observe:Counter localCounter = new("local_operations");
+        localCounter.increment();
+        //Increment the value of the counter by 20.
+        localCounter.increment(amount = 20);
+
+        //Create a counter with optional fields description and tags.
+        observe:Counter registeredCounter = new("product_total_product_order_quantity",
+            desc = "Total quantity required", tags = {prodName:"HeadPhone", prodType:"Electronics"});
+
+        //Register the counter instance, therefore it is stored in the global registry and can be reported to the
```

Comment:
Can we change to something like 'Register the counter instance and thereby store it in the global registry. It can be reported...'?